### PR TITLE
Proposed additions to readme file for more information and clarity on…

### PR DESCRIPTION
**Pull request by zsadler**
… what elasticsearch can do and be used for

To make Elasticsearch more approachable to a larger number of users—from informed developers to curious passersby and new and interested visitors—I have added a new section to the Elasticsearch README. The section is entitled What to Use Elasticsearch for and When to Use It. As the name suggests, the new section, a brief paragraph, explains in plain and straightforward language what Elasticsearch is used for and when one should use it.

### CODE REVIEW

File: README.textile [hunk 1]
```diff
@@ -1,5 +1,20 @@
 h1. Elasticsearch
 
+h2. What to Use Elasticsearch For and When to Use It.
+
+Have extremely large amounts of data to churn through? Want to return searches quickly and accurately? Elasticsearch can easily index massive amounts of data and return results faster with less resource consumption when compared to other search technologies (e.g. conventional SQL database search).
```
Comment (dakrone) [inline README.textile]: This line is okay, but I'd prefer to remove any mention of other systems, perhaps:

"Have extremely large amounts of data to churn through? Want to return searches quickly and accurately? Elasticsearch can easily index massive amounts of data and return results extremely fast."

**Comment by dakrone** [comment]
Left a comment about this, if you want to address that I'll be happy to merge it.

**Comment by zsadler** [comment]
Made the suggested change and pushed it up. Thanks for the merge!

**Comment by jasontedor** [comment]
I have the same thoughts I had on #16595. I'm not sure I see the point, I don't like how it displaces more valuable content down the page, and many of the word-choices are non-specific.

**Comment by zsadler** [comment]
Hi Jason,

I see your point but as a developer myself I'd heard of ElasticSearch from a backend dev who was adding it to a blog site we built at work. Wanting to understand what it was I came here to check it out. After I read the readme I found myself saying ok but what are other use cases? Now I'm totally new to this type of thing, being for the a frontend dev, and I had to do more research to figure out how this could actually be utilized. Call me a novice if you must but if there's one there must be more. By the other users post it seems he's trying to add clarity as well. Maybe there might be a need for this for people who don't have your level of understanding. I hope this clears up why I felt there was a need to add more.

**Comment by jasontedor** [comment]
Sure, but this is just displacing a link to the [product page](https://www.elastic.co/products/elasticsearch) that already includes such detail and even includes a link to [use cases](https://www.elastic.co/use-cases)? Keep in mind that the GitHub repository is oriented towards developers.

**Comment by clintongormley** [comment]
I'm tending to agree with @jasontedor here

**Comment by jasontedor** [comment]
Thanks for the pull request @zsadler, but I think we prefer the README the way that it is currently written, leaving the product pages and use cases pages on the Elastic website for the content that you were looking to contribute here.

**Pull request closed without merging**